events/schema.py

```diff
@@ -61,19 +65,33 @@
 EventGroupTranslation = apps.get_model("events", "EventGroupTranslation")
 
 
-def validate_enrolment(child: Child, occurrence: Occurrence):
+def validate_enrolment(child: Child, occurrence: Occurrence) -> None:
     if reason := occurrence.get_enrolment_denied_reason(child):
         raise ENROLMENT_DENIED_REASON_TO_GRAPHQL_ERROR[reason]
 
 
-def validate_enrolment_deletion(enrolment):
+def validate_enrolment_deletion(enrolment: Enrolment) -> None:
+    now = timezone.now()
+    occurrence_time = enrolment.occurrence.time
+
+    # Check if the enrolment is for an upcoming occurrence.
+    # User should never be able to unenrol, if the occurrence is in the past.
     if not enrolment.is_upcoming():
         raise PastEnrolmentError(
             "Cannot unenrol from an occurrence that is in the past."
         )
 
+    # Get the unenrol hours limit from settings, 0 means no time limit
+    # (only past occurrences are not allowed to unenrol).
+    hours_before = getattr(settings, "KUKKUU_ENROLMENT_UNENROL_HOURS_BEFORE", 48)
```

Comment:
We can trust the settings default value and let it fail / error if it for some reason is None